### PR TITLE
docs: update notebooks - bring back fabric reviewers changes.

### PR DESCRIPTION
**Pull request by JessicaXYWang**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

File: notebooks/features/classification/Classification - Before and After SynapseML.ipynb [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+   "metadata": {
+    "tags": [
+     "synapse_internal_only"
+    ]
+   },
+   "source": [
+    "## Prerequisites\n",
     "\n",
-    "<p><img src=\"https://images-na.ssl-images-amazon.com/images/G/01/img16/books/bookstore/landing-page/1000638_books_landing-page_bookstore-photo-01.jpg\" style=\"width: 500px;\" title=\"Image from https://images-na.ssl-images-amazon.com/images/G/01/img16/books/bookstore/landing-page/1000638_books_landing-page_bookstore-photo-01.jpg\" /><br /></p>\n",
+    "* Attach your notebook to a lakehouse. On the left side, select **Add** to add an existing lakehouse or create a lakehouse."
```
Comment (mhamilton723) [inline notebooks/features/classification/Classification - Before and After SynapseML.ipynb:22]: Lets factor this out into a common installation and setup doc so its not in every NB

File: notebooks/features/classification/Classification - Sentiment Analysis Quickstart.ipynb [hunk 2]
```diff
@@ -11,7 +11,32 @@
     }
    },
    "source": [
-    "# A 5-minute tour of SynapseML"
+    "# Your First SynapseML Model\n",
+    "This tutorial provides a brief introduction to building your first machine learning model using SynapseML, demonstrating how SynapseML makes it easy to do complex machine learning tasks. We use SynapseML to create a small ML training pipeline with a featurization stage and LightGBM regression stage to predict ratings based on review text from a dataset containing book reviews from Amazon. Finally we showcase how SynapseML makes it easy to use prebuilt models to solve problems without having to re-solve them yourself."
```
Comment (mhamilton723) [inline notebooks/features/classification/Classification - Sentiment Analysis Quickstart.ipynb:15]:
```suggestion
    "This tutorial provides a brief introduction to SynapseML. In particular, we use SynapseML to create two different pipelines for sentiment analysis. The first pipeline combines a text featurization stage with LightGBM regression to predict ratings based on review text from a dataset containing book reviews from Amazon. The second pipeline shows how to use prebuilt models through the Azure Cognitive Services to solve this problem without training data."
```

File: notebooks/features/cognitive_services/CognitiveServices - Create a Multilingual Search Engine from Forms.ipynb [hunk 3]
```diff
@@ -17,7 +18,7 @@
    "source": [
     "# Tutorial: Create a custom search engine and question-answering system\n",
     "\n",
-    "In this tutorial, learn how to index and query large data loaded from a Spark cluster. You'll set up a Jupyter Notebook that performs the following actions:\n",
+    "In this tutorial, learn how to index and query large data loaded from a Spark cluster. You set up a Jupyter Notebook that performs the following actions:\n",
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - Create a Multilingual Search Engine from Forms.ipynb:21]:
```suggestion
    "In this tutorial, learn how to index and query large data loaded from a Spark cluster. You'll set up a Jupyter Notebook that performs the following actions:\n",
```

File: notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb [hunk 4]
```diff
@@ -139,31 +174,32 @@
...
     "anomaly_loc = \"westus2\"\n",
     "\n",
     "# A Translator subscription key\n",
-    "translator_key = find_secret(\"translator-key\")\n",
+    "translator_key = find_secret(\"translator-key\") # Replace it with your cognitive service key, check prerequisites for more details\n",
     "translator_loc = \"eastus\"\n",
     "\n",
     "# An Azure search key\n",
-    "search_key = find_secret(\"azure-search-key\")"
+    "search_key = find_secret(\"azure-search-key\") # Replace it with your cognitive service key, check prerequisites for more details"
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb:192]: Maybe make this warning something clearer like 

# Replace the call to find_secret with your key as a python string. e.g. key="27snaiw..."

File: notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb [hunk 5]
```diff
@@ -82,10 +88,35 @@
...
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "# Bootstrap Spark Session\n",
+    "spark = SparkSession.builder.getOrCreate()\n",
+    "\n",
+    "from synapse.ml.core.platform import *\n",
+    "\n",
+    "if running_on_synapse():\n",
+    "    shell = TerminalInteractiveShell.instance()\n",
+    "    shell.define_macro(\"foo\", \"\"\"a,b=10,20\"\"\")\n",
```
Comment (mhamilton723) [inline notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb:114]:
```suggestion
```

File: notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb [hunk 6]
```diff
@@ -576,7 +629,10 @@
      "nuid": "ff8345cf-fdbc-482d-bf92-5d560724d245",
      "showTitle": false,
      "title": ""
-    }
+    },
+    "tags": [
+     "hide-synapse-internal"
```
Comment (mhamilton723) [inline notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb:634]: this notebook is using this alot, is there any way to reduce the usage of these here?

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Lovely work, left a few comments!

File: notebooks/features/classification/Classification - Before and After SynapseML.ipynb [hunk 7]
```diff
@@ -159,7 +192,7 @@
     "   `test` dataset\n",
     "5. Evaluate the best model on the `validation` set\n",
     "\n",
-    "As you can see below, there is a lot of work involved and a lot of\n",
+    "As you can see, there's numerous work involved and many\n",
```
[the remark this answers is not in the document]
Comment (JessicaXYWang) [reply notebooks/features/classification/Classification - Before and After SynapseML.ipynb]: removed by fabric reviewer, can we delete it here as well?

File: notebooks/features/classification/Classification - Before and After SynapseML.ipynb (hunk 1 above)
Comment (JessicaXYWang) [reply notebooks/features/classification/Classification - Before and After SynapseML.ipynb]: move it to manifest file to make it platform independent.

File: notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb (hunk 6 above)
Comment (JessicaXYWang) [reply notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb:619]: Part of this notebook cannot run successfully on Fabric. Will remove the exclusion after I address that issue.

File: notebooks/features/cognitive_services/CognitiveServices - Multivariate Anomaly Detection.ipynb [hunk 8]
```diff
@@ -687,10 +303,11 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's now format the `interpretation` column that stores the contribution score from each sensor to the detected anomalies. The next cell formats this data, and splits the contribution score of each sensor into its own column."
+    "Let's now formatted the `contributors` column that stores the contribution score from each sensor to the detected anomalies. The next cell formats this data, and splits the contribution score of each sensor into its own column."
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - Multivariate Anomaly Detection.ipynb:310]:
```suggestion
    "Let's now format the `contributors` column that stores the contribution score from each sensor to the detected anomalies. The next cell formats this data, and splits the contribution score of each sensor into its own column."
```

File: notebooks/features/lightgbm/LightGBM - Overview.ipynb [hunk 9]
```diff
@@ -56,33 +37,33 @@
     "\n",
     "### LightGBM Usage:\n",
     "\n",
-    "- LightGBMClassifier: used for building classification models. For example, to predict whether a company will bankrupt or not, we could build a binary classification model with LightGBMClassifier.\n",
+    "- LightGBMClassifier: used for building classification models. For example, to predict whether a company enters bankrupts or not, we could build a binary classification model with LightGBMClassifier.\n",
```
Comment (mhamilton723) [inline notebooks/features/lightgbm/LightGBM - Overview.ipynb:40]:
```suggestion
    "- LightGBMClassifier: used for building classification models. For example, to predict whether a company enters bankruptcy or not, we could build a binary classification model with LightGBMClassifier.\n",
```

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Just some minor nits

File: notebooks/features/classification/Classification - Before and After SynapseML.ipynb [hunk 10]
```diff
@@ -235,16 +256,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 4b. Classify using synapseml\n",
+    "## Classify using SynapseML\n",
     "\n",
-    "Life is a lot simpler when using `synapseml`!\n",
+    "The steps needed with `synapseml` are simpler:\n",
```
Comment (mhamilton723) [inline notebooks/features/classification/Classification - Before and After SynapseML.ipynb:261]:
```suggestion
    "The pipeline can be simplified by using SynapseML:\n",
```

File: notebooks/features/responsible_ai/Interpretability - Tabular SHAP explainer.ipynb [hunk 11]
```diff
@@ -308,17 +313,21 @@
...
      "title": ""
-    }
+    },
+    "tags": [
+     "hide-synapse-internal"
+    ]
    },
    "source": [
-    "Your results will look like:\n",
+    "Your results look like:\n",
```
Comment (mhamilton723) [inline notebooks/features/responsible_ai/Interpretability - Tabular SHAP explainer.ipynb:330]:
```suggestion
    "Your results should look like:\n",
```

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**